client/state/timezones/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import { TIMEZONES_RECEIVE } from 'state/action-types';
+
+import {
+	rawOffsetsSchema,
+	labelsSchema,
+	continentsSchema
+} from './schema';
+
+export const rawOffsets = createReducer( {}, {
+	[ TIMEZONES_RECEIVE ]: ( state, actions ) => ( { rawOffsets: actions.rawOffsets } )
```

Comment:
It looks like we're doubling up on labels. We should be able to just return the actions.rawOffsets instead. The same goes for the other reducers below
```
state.timezones.rawOffsets.rawOffsets.
```

<img width="199" alt="screen shot 2017-02-07 at 10 25 16 am" src="https://cloud.githubusercontent.com/assets/1270189/22705275/c37fcf16-ed1f-11e6-8084-01986b661f41.png">